online-shop/src/components/product/product.component.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import useProductStyles from "./product.styles"
+import { Button,Grid} from "@material-ui/core";
+import Typeograpy from "@material-ui/core/Typography"
+import { Link } from "react-router-dom";
+
+
+export interface Produs {
```

Comment:
Don't name stuff in romanian, please. I would rather you name this Product and the component below ProductComponent, for sake of clarity.